File: src/main/java/UserPVLogSchema.java
import org.apache.flink.streaming.util.serialization.SerializationSchema;


public class UserPVLogSchema implements SerializationSchema<UserPVTraceLog> {
    @Override
    public byte[] serialize(UserPVTraceLog userPVTraceLog) {
        //TODO
        return new byte[0];
    }
}

File: src/main/scala/UserPVLogSchema.scala
import org.apache.flink.streaming.util.serialization.SerializationSchema

/**
  * Created by whjiang on 2016/12/22.
  */
class UserPVLogSchema extends SerializationSchema[UserPVTraceLog] {
  override def serialize(t: UserPVTraceLog): Array[Byte] = {
    null
  }
}
